### ISSUE TITLE
Keyboard shortcuts not working in macOS Tahoe 26.0

### ISSUE THREAD

**Issue by ATXMJ** [rxhanson/Rectangle#1648]
I can't seem to get keyboard shortcuts working on macOS Tahoe 26.0.

The Rectangle app never asks to be granted Accessibility or Input Monitoring permissions. Rectangle does not appear in the list of applications for either permission (under "Privacy & Security" > "Accessibility" or "Input Monitoring".

I've tried everything mentioned in the [Troubleshooting](https://github.com/rxhanson/Rectangle?tab=readme-ov-file#troubleshooting) section of the README, as well as everything else I can think of, including manually adding Rectangle to those permission lists, restarting Rectangle and macOS, reinstalling Rectangle, etc etc etc.

Rectangle does detect my keyboard input for customizing shortcuts in the settings, but I assume that doesn't rely on the accessibility permissions.

Any ideas?

 
- [✓] This is a new bug that hasn't been documented in a closed issue or in the Readme.

macOS version: Tahoe 26.0
Rectangle version: 0.90
Logs if applicable (In Rectangle menu, hold option, "View Logging..."):

**Comment by rxhanson** [comment]
Try resetting the Accessibility permissions via Terminal, and you should be prompted to re-add them when it relaunches:

```
killall Rectangle
tccutil reset All com.knollsoft.Rectangle
open -a Rectangle
```

**Comment by ehmjayy** [comment]
Tried that before and just tried it again with no luck. Rectangle just starts normally (in the background / menu bar) without any prompts.

**Comment by rxhanson** [comment]
Let's try adding it manually: In System Settings > Privacy & Security > Accessibility, click on the plus button at the bottom and select Rectangle. Then restart Rectangle.

**Comment by ehmjayy** [comment]
Okay, I swear I tried that before with no luck, but I just tried again and it seems to have worked.

Consider this resolved!
